src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,146 @@
-import { PeopleFilters } from './PeopleFilters';
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { Person } from '../types/Person';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { PeopleFilters } from './PeopleFilters';
+import { SearchParam } from '../types/SearchParam';
+
+function getFilteredPeople(
+  people: Person[],
+  sex: string | null,
+  query: string | null,
+  centuries: string[],
+  sortField: string | null,
+  order: string | null,
+): Person[] {
+  let preparedPeople = [...people];
+
+  if (sex) {
+    preparedPeople = preparedPeople.filter(person => person.sex === sex);
+  }
+
+  if (centuries.length > 0) {
```

Comment:
```suggestion
  if (!!centuries.length) {
```

Reply:
I can't type !! before centuries.length because ESlint show error "Redundant double negation.eslint". So in the next commit I'll type just  centuries.length